docs/content/querying/filters.md

```diff
@@ -133,6 +147,9 @@ Search filters can be used to filter on partial string matches.
 |dimension|The dimension to perform the search over.|yes|
 |query|A JSON object for the type of search. See below for more information.|yes|
```

Comment:
would also add an `extractionFn` item here (and actually I don't think the block below is needed if you have this; that content would work in this table)